applications/luci-app-wireguard/luasrc/view/wireguard.htm

```diff
@@ -182,6 +182,18 @@
 <h2>WireGuard Status</h2>
```

Comment:
@he-ma I think he means that

Reply:
Well, but this is not newly indroduced by me and has been in the original ever since.